docs/authentication/social-connections/slack.mdx

```diff
@@ -30,10 +30,10 @@ First, you need to create a new OAuth Slack app. On the main [Slack apps page](h
 
 ![Creating a new application](/docs/images/authentication-providers/slack/creating-new-application.jpg)
 
-In the Clerk Dashboard, go to **User & Authentication > [Social Connections](https://dashboard.clerk.com/last-active?path=user-authentication/social-connections)** and enable Slack. In the modal that opened, ensure **Use custom credentials** is enabled and copy **Authorized redirect URI**. Navigate to **OAuth & Permissions > Redirect URLs** and paste the value to add a new record.
+Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections). In the top navigation, select **Configure**. Then in the sidebar, select **SSO Connections**. Select the **Add connection** button, and select **For all users**. In the **Choose provider** dropdown, select **Slack**. Toggle on **Use custom credentials** and copy **Authorized redirect URI**. Navigate to **OAuth & Permissions > Redirect URLs** and paste the value to add a new record.
 
 ![Copying values from the Slack dashboard](/docs/images/authentication-providers/slack/copying-values-from-slack-dashboard.jpg)
 
 Copy the **Client ID** and **Client secret** as shown in the above image from the Basic Information menu of your app. Go back to the Clerk Dashboard and paste them into the respective fields.
 
-Don't forget to click **Apply** in the Clerk dashboard. Congratulations! Social connection with Slack is now configured for your instance.
+Finally, click **Add connection** so that the settings are applied. Congratulations! Social connection with Slack is now configured for your instance.
```

Comment:
```suggestion
Finally, select **Add connection** so that the settings are applied. Congratulations! Social connection with Slack is now configured for your instance.
```